.github/workflows/hatch_ci.yml

```diff
@@ -13,13 +13,39 @@ on:
         required: false
         default: "3.12"
         description: "Maximum Python version to test against."
+      whitesource_product_name:
+        type: string
+        required: false
+        description: "WhiteSource product name"
+      whitesource_project_name:
+        type: string
+        required: false
+        default: ${{ github.event.repository.name }}
```

Comment:
Defaulting the repo name here, but the product name should be coming from the repository itself to map to the correct product.